todo-app/src/stores/TodoStore.js

```diff
@@ -0,0 +1,54 @@
+import { action, observable } from 'mobx';
+import moment from 'moment';
+
+// Data
+import TodoData from '../data/todo.json';
+
+export class TodoStore {
+  @observable todos = TodoData
+  @observable newDescription = ''
+  @observable newDeadline = moment().format('YYYY-MM-DD')
```

Comment:
You're setting the default todo deadline to today. Are you sure this is what you want to do? Why not leave it blank?